app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/view/BottomBarBehavior.kt

```diff
@@ -0,0 +1,35 @@
+package io.github.droidkaigi.confsched2018.presentation.common.view
+
+import android.content.Context
+import android.support.constraint.ConstraintLayout
+import android.support.design.widget.AppBarLayout
+import android.support.design.widget.CoordinatorLayout
+import android.util.AttributeSet
+import android.view.View
+
+/**
+ * A custom view for showing move next or prev session
+ *
+ * ref: https://github.com/DroidKaigi/conference-app-2018/issues/180
+ * Created by hakutosuzuki on 2018/01/18.
```

Comment:
Could you remove `Created by`? It remains to git 😄

Reply:
sorry ;p
fix it!